src/Illuminate/Foundation/Http/MaintenanceModeBypassCookie.php

```diff
@@ -20,7 +20,7 @@ public static function create(string $key)
         return new Cookie('laravel_maintenance', base64_encode(json_encode([
             'expires_at' => $expiresAt->getTimestamp(),
             'mac' => hash_hmac('SHA256', $expiresAt->getTimestamp(), $key),
-        ])), $expiresAt, config('session.path'), config('session.domain'), config('session.secure'));
+        ])), $expiresAt, '/', env('SESSION_DOMAIN', null));
```

Comment:
The env helper cannot be used, ever. In production, the env files should not be loaded, in favour of config caching.